neo/app/controllers/Application.scala

```diff
@@ -59,7 +60,8 @@ class Application @Inject()(val wsClient: WSClient,
   }
 
   def ws = WebSocket.accept[JsValue, JsValue] { request =>
-    ActorFlow.actorRef(out => NeoActor.props(out, wsClient, request.host, config))
+    val prop = BerryClient.props(new JSONParser(), manager, new QueryPlanner(), config)
+    ActorFlow.actorRef(out => RequestRouter.props(out, prop, config))
```

Comment:
If we give `BerryClient` an `out` in `prop`, it will send the response directly to `out`, not the `sender()`, namely RequestRouter.